packages/backend-function/src/function_env_type_generator.ts

```diff
@@ -69,10 +77,15 @@ export class FunctionEnvironmentTypeGenerator {
   }
 
   /**
-   * Generate an un-typed process.env shim
+   * Generate an any-typed process.env shim if doesn't exist
    */
   generateProcessEnvShim = () => {
-    const content = `${this.header}${EOL}${this.envAssignment};`;
+    // Don't recreate the file
```

Comment:
we need to recreate this file. Reason is if the customer has added an env var and used it in their handler definition that would fail typechecking until the type can be regenerated

Reply:
yeah the edge case would be where the synth would fail and then TSC would complain about the new env var not available. Will update.